home/utils.py

```diff
@@ -217,86 +248,338 @@ def add_viber(row, page=None):
             print(f"Content page not created for {row}")
 
 
-def export_content_csv():
-    fieldnames = [
-        "parent",
-        "web_title",
-        "web_subtitle",
-        "web_body",
-        "whatsapp_title",
-        "whatsapp_body",
-        "messenger_title",
-        "messenger_body",
-        "viber_title",
-        "viber_body",
-        "image_name",
-        "translation_tag",
-        "tags",
-        "quick_replies",
-        "triggers",
-        "locale",
-    ]
-
-    def write_rows(rows):
-        writer = csv.DictWriter(
-            sys.stdout,
-            fieldnames=fieldnames,
+def get_messages(platform_body: blocks.StreamValue) -> List[dict]:
+    """Formats the platform body objects into a list of dictionaries"""
+    msgs = []
+    for msg in platform_body:
+        msgs.append(
+            {
+                "msg": msg.value["message"] if "message" in msg.value else None,
+                "img": msg.value["image"] if "image" in msg.value else None,
+                "doc": msg.value["document"] if "document" in msg.value else None,
+                "med": msg.value["media"] if "media" in msg.value else None,
+                "next_prompt": msg.value["next_prompt"]
+                if "next_prompt" in msg.value
+                else None,
```

Comment:
Could also be `msg.get("next_prompt")`, or `msg.get("next_prompt", default=None)` if you want to be more explicit. But works either way. (assuming that this `msg` thing acts like a dictionary)

Reply:
Unfortunately there is no get method :( I initially did try and eventually found this 

https://groups.google.com/g/wagtail/c/a6tA-ftoYD4/m/7s3Pm9gDBwAJ

Reply:
Ah I see, so it doesn't actually act like a dictionary. Yeah happy to leave it like this, maybe add a comment as to why we're doing it like this.